app/views/split_checkout/payment/_stripe_sca.html.haml

```diff
@@ -1,4 +1,4 @@
-%div{"data-controller": "stripe-cards"}
+%div{"data-controller": "stripe-cards", "data-paymentid": "#{payment_method.id}" }
```

Comment:
As this is the value of `payment_method.id`, I suggest the variable should include the word "method". 

But what format to use for multi-word 'data-' attributes? It looks like we use hyphen-separated-words throughout the repo, so to be consistent `data-payment-method-id`. Which I guess gets converted to `paymentMethodId` in JS.

Reply:
Oh, and we already have `data-paymentmethod-id` elsewhere, which I guess is different? Can these be unified? The both seem to refer to the underlying `payment_method.id`, so I think they should be named the same.

Reply:
Thanks!

 - I renamed the value with commit: 3247c32191 to be more consistant. 
 - As far as I know, we can't use the same data attribute, but to be more consistant as well, I delete the `paymentmethod` string previously appended. See commit 251ab2ac9a.

I think it's correct now

Reply:
Great, that looks simpler!